### PR TITLE
Windows metric brushup

### PR DESCRIPTION
**Pull request by daiksy**

### CODE REVIEW

File: util/util_windows.go [hunk 1]
```diff
@@ -26,3 +29,47 @@ func RunCommand(command string) (string, string, error) {
...
+type FilesystemInfo struct {
+	PercentUsed string
+	KbUsed      float64
+	KbSize      float64
+	KbAvailable float64
+	Mount        string
+	Label        string
+	VolumeName  string
+	FsType      string
+}
```
Comment (mechairoi) [inline util/util_windows.go]: [IMHO] `type FilesystemInfo` is only used by `util/windows/filesystem`. I think it's better to move to `util/windows/filesystem.go`.

File: metrics/windows/disk.go [hunk 2]
```diff
@@ -106,7 +106,7 @@ func (g *DiskGenerator) Generate() (metrics.Values, error) {
 	results := make(map[string]float64)
 	for _, v := range g.counters {
 		var value windows.PDH_FMT_COUNTERVALUE_ITEM_DOUBLE
-		r, _, err := windows.PdhGetFormattedCounterValue.Call(uintptr(v.Counter), windows.PDH_FMT_DOUBLE, uintptr(0), uintptr(unsafe.Pointer(&value)))
+		r, _, err := windows.PdhGetFormattedCounterValue.Call(uintptr(v.Counter), windows.PDH_FMT_DOUBLE, uintptr(0), uintptr(unsafe.Pointer(&value.FmtValue)))
```
Comment (mechairoi) [inline metrics/windows/disk.go]: It looks like `PDH_FMT_COUNTERVALUE_ITEM_DOUBLE` is not used.
I think it is better to use `var value PDH_FMT_COUNTERVALUE_DOUBLE` and to get rid of all `PDH_FMT_COUNTERVALUE_ITEM_DOUBLE`.

File: metrics/windows/interface.go [hunk 3]
```diff
@@ -53,6 +53,7 @@ func NewInterfaceGenerator(interval time.Duration) (*InterfaceGenerator, error)
 				name := windows.BytePtrToString(&ai.Description[0])
 				name = strings.Replace(name, "(", "[", -1)
 				name = strings.Replace(name, ")", "]", -1)
+				name = strings.Replace(name, "#", "_", -1)
```
Comment (mechairoi) [inline metrics/windows/interface.go]: :expressionless:

**Comment by mattn** [comment]
LGTM, Thanks for correcting.

**Comment by mechairoi** [comment]
LGTM, too.

File: util/util_windows.go (hunk 1 above)
Comment (daiksy) [inline util/util_windows.go]: I think so too!

File: metrics/windows/disk.go (hunk 2 above)
Comment (daiksy) [inline metrics/windows/disk.go]: You 're right!

**Comment by mechairoi** [comment]
LGTM :space_invader:

**Pull request merged**